client/src/Tickets/Tickets.jsx

```diff
@@ -75,24 +76,7 @@ Filters.propTypes = {
 
 const Tickets = () => {
   const [columnFilters, setColumnFilters] = useState([]);
-  const [toggleUserModal, setToggleUserModal] = useState(false);
   const [data, setData] = useState([]);
-  const { user } = useAuthContext();
-  const [ticket, setTicket] = useState({
```

Comment:
It was really confusing having both the data for the list/table AND the data for the modals in this component. Now, we only have the data for the list/table (which could also probably be moved into TicketTable?)...